Jenkinsfile

```diff
@@ -86,6 +86,25 @@ pipeline {
                         }
                     }
                 }
+                stage('external plugins') {
```

Comment:
Can we get rid of this new stage, as we are now using the same job as for bundled plugins?